cmd/releasego/generate-announcement_test.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestGenerateAnnouncement(t *testing.T) {
+
```

Comment:
I think it would be good to include a golden test like `Test_generateContent` in `cmd\releasego\add-retry-instructions_test.go`. It's fairly simple as far as tests go (just a baseline test) but it really helps me a lot in the dev process, and it helps PR review too because it's really easy to see some examples.

Reply:
For the golden tests, this is what I mean: https://github.com/microsoft/go-infra/commit/4c79656cc30ce288211eee138e1ed554e458754c (the goldentest package has some comments about what's going on). George thinks he marked this comment as resolved by mistake, so just adding some more context while I'm here reopening it. 🙂

Reply:
Added :)